data/blog/2024-01-31-vector-announcement.mdx

```diff
@@ -0,0 +1,198 @@
+---
+slug: introducing-vector-database
+title: "Upstash Vector: Serverless Vector Database for AI and LLMs"
+authors:
+  - mehmet
+tags: [serverless, database, vector, announcement, ai, llm]
+---
+
+Today, we're introducing Upstash Vector, a serverless vector database built for working with vector embeddings for AI models and LLMs. Upstash Vector will help you to efficiently store and query high-dimensional vector embeddings. Upstash Vector operates on a serverless model, abstracting away hosting and management complexities.
```

Comment:
Suggested: Upstash Vector will help you efficiently store and query high-dimensional vector embeddings on a serverless model, eliminating the hosting and management complexities.